pkg/server/server.go

```diff
@@ -420,6 +477,20 @@ func NewServerWithOpts(opts ...OpenFGAServiceV1Option) (*Server, error) {
 		}
 	}
 
+	if s.listObjectsDispatchThrottlingEnabled {
+		s.logger.Info("Enabling ListObjects dispatch throttling",
+			zap.Duration("Frequency", s.listObjectsDispatchThrottlingFrequency),
+			zap.Uint32("DefaultThreshold", s.listObjectsDispatchDefaultThreshold),
+			zap.Uint32("MaxThreshold", s.listObjectsDispatchThrottlingMaxThreshold),
+		)
+
+		if s.listObjectsDispatchThrottlingMaxThreshold != 0 && s.listObjectsDispatchDefaultThreshold > s.listObjectsDispatchThrottlingMaxThreshold {
+			return nil, fmt.Errorf("ListObjects default dispatch throttling threshold must be equal or smaller than max dispatch threshold for ListObjects")
+		}
+
```

Comment:
Do we need any unit test that exercise listObject at server.go level for throttling?  If not, that's probably ok but want to double check

Reply:
I think we already have one for that `TestServerPanicIfDefaultListObjectThresholdGreaterThanMaxDispatchThreshold`